python/paddle/fluid/tests/unittests/ir/inference/test_conv_int8_scales_pass.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from auto_scan_test import PassAutoScanTest, SkipReasons
```

Comment:
Please check if you need all imports in this file

Reply:
It works correctly without `SkipReasons`, but I see most of the other tests have these either `IgnoreReasons` included (as well as other imports so I followed the pattern)

Reply:
![image](https://user-images.githubusercontent.com/17185347/165729259-92ed3653-51d8-485b-8f9c-5aa866a880dd.png)
You should be able to remove all darkened (unused)